Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -31,8 +31,14 @@ function OnUpdate_GasGenerator( furniture, deltaTime )
         end
     end
 	return
+	furniture.SetAnimationState("running")
 end
 
+function OnPowerOff_GasGenerator( furniture, deltaTime )
```

Comment:
I'm not sure... we have half one and half the other. I think I agree with you - I was just trying to be consistent